integration-cli/docker_cli_network_unix_test.go

```diff
@@ -523,6 +523,22 @@ func (s *DockerNetworkSuite) TestDockerNetworkCustomIpam(c *check.C) {
 	assertNwNotAvailable(c, "br0")
 }
 
+func (s *DockerNetworkSuite) TestDockerNetworkIpamOptions(c *check.C) {
+	// Create a bridge network using custom ipam driver and options
+	dockerCmd(c, "network", "create", "--ipam-driver", dummyIpamDriver, "--ipam-opt", "opt1=drv1", "--ipam-opt", "opt2=drv2", "br0")
+	assertNwIsAvailable(c, "br0")
+
+	// Verify expected network ipam options
+	nr := getNetworkResource(c, "br0")
+	opts := nr.IPAM.Options
+	c.Assert(opts["opt1"], checker.Equals, "drv1")
+	c.Assert(opts["opt2"], checker.Equals, "drv2")
+
+	// remove network
```

Comment:
No, just this removal bit, the test suite will take care of removing the network.